### PR TITLE
added open pilot

### PR DESCRIPTION
**Pull request by falkorichter**

### CODE REVIEW

File: Casks/open-pilot.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'open-pilot' do
+  version '15.02.01'
+  sha256 '21f1db27188ad05cc755fdcdda97853bdd6ba6adfc09afe78095c414065d7da5'
+
+  url 'http://bit.ly/1xVQJNA'
+  name 'OpenPilot Ground Control Station'
+  homepage 'https://wiki.openpilot.org/display/WIKI/OpenPilot+Downloads'
+  license : "GPLv3"
```
Comment (vitorgalvao) [inline Casks/open-pilot.rb]: `license 'GPLv3'`

File: Casks/open-pilot.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'open-pilot' do
+  version '15.02.01'
+  sha256 '21f1db27188ad05cc755fdcdda97853bdd6ba6adfc09afe78095c414065d7da5'
+
+  url 'http://bit.ly/1xVQJNA'
```
Comment (vitorgalvao) [inline Casks/open-pilot.rb]: `url "https://wiki.openpilot.org/download/attachments/361104/OpenPilot-RELEASE-#{version}-osx.dmg"`

File: Casks/open-pilot.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'open-pilot' do
```
Comment (vitorgalvao) [inline Casks/open-pilot.rb]: Token is wrong, it needs to follow the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_token_reference.md) (`openpilot-gcs`).

**Comment by vitorgalvao** [comment]
Please [squash your commits](http://davidwalsh.name/squash-commits-git) after making the requested changes.

**Comment by fanquake** [comment]
@falkorichter Do you plan on updating this?

**Comment by falkorichter** [comment]
Feedback applied and squashed :+1:

File: Casks/openpilot-gcs.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'openpilot-gcs' do
+  version '15.02.01'
+  sha256 '21f1db27188ad05cc755fdcdda97853bdd6ba6adfc09afe78095c414065d7da5'
+
+  url "https://wiki.openpilot.org/download/attachments/361104/OpenPilot-RELEASE-#{version}-osx.dmg"
+  name 'OpenPilot Ground Control Station'
+  homepage 'https://wiki.openpilot.org/display/WIKI/OpenPilot+Downloads'
+  license : "GPLv3"
```
Comment (vitorgalvao) [inline Casks/openpilot-gcs.rb]: Again, this is wrong. It has to be `license :gpl`.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in the below referenced pull request. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**